### PR TITLE
Create RC which includes latest versions of Setzer, Omnia & Oracle Suite tools

### PR DESCRIPTION
**Pull request by teghnet**
## Merge Criteria

- [ ] run `nix-env -i -f https://github.com/makerdao/oracles-v2/tarball/stable` and then run `nix-env -i -f .`
- [ ] install `omnia` as a *feed* on a clean VPS using `install-omnia`
  - [ ] make sure `systemctl status omnia` is `active` and hasn't exited
  - [ ] check that output from `journalctl -e -u omnia` looks reasonable
  - [ ] make sure `systemctl status ssb-server` is `active` and hasn't exited
  - [ ] check that output from `journalctl -e -u ssb-server` looks reasonable
  - [ ] reboot VPS and make sure services auto-start
- [ ] install `omnia` as a *relay* on a clean VPS using `install-omnia`
  - [ ] make sure `systemctl status omnia` is `active` and hasn't exited
  - [ ] check that output from `journalctl -e -u omnia` looks reasonable
  - [ ] make sure `systemctl status ssb-server` is `active` and hasn't exited
  - [ ] check that output from `journalctl -e -u ssb-server` looks reasonable
  - [ ] reboot VPS and make sure services auto-start

### CODE REVIEW

File: systemd/install-omnia [hunk 1]
```diff
@@ -97,7 +106,6 @@ copyOmniaConf() {
   fi
 
 	configUpdates+=( ".options.logFormat = \"text\"" )
-	configOverrides+=( ".options.setzerEthRpcUrl = \"http://127.0.0.1:9989\"" )
```
Comment (madumas) [inline systemd/install-omnia:100]: why remove this? with line 323, this is disabling splitter-agent for setzer

File: systemd/install-omnia [hunk 2]
```diff
@@ -46,65 +54,72 @@ Options:
...
-  envsubst '
+	local  name
+	name="$1.service"
+	local  target
+	target="$TARGET_DIR"/"$name"
+	local  restart
+	restart=$(               if [[ -e "$target" ]]; then echo 1; fi)
+
+	# shellcheck disable=SC2016
+	envsubst  '
```
Comment (jar-o) [inline systemd/install-omnia:69]: not sure I understand what's going on here

Comment (teghnet) [reply systemd/install-omnia:69]: We do not want the expression to expand here, but to make `envsubst` substitute the listed env vars. 
To list the env vars we apparently need a normal shell notation i.e. `$var`

Comment (jar-o) [reply systemd/install-omnia:69]: ah, I see. bash witchery

Comment (teghnet) [reply systemd/install-omnia:69]: :magic_wand:

**Pull request merged**